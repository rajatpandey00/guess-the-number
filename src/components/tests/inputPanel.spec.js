import React from "react";
import renderer from "react-test-renderer";
import InputPanel from "../inputPanel";

describe("InputPanel test", () => {
	test("snapshot renders", () => {
		const component = renderer.create(<InputPanel/>);
		let tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});

