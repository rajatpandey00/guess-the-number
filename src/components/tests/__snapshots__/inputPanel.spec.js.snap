// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputPanel test snapshot renders 1`] = `
<div>
  <h1>
    Guess The Number Game
  </h1>
  <div>
    <div
      style={
        Object {
          "display": "inline-flex",
        }
      }
    >
      <h1>
        Hint
      </h1>
      <div
        id="game_hint"
      >
        
      </div>
    </div>
  </div>
  <div
    contentEditable={true}
    id="game_input"
    onInput={[Function]}
  />
  <br />
  <button
    onClick={[Function]}
  >
    Submit
  </button>
  
</div>
`;
